Source/WebCore/mathml/MathMLRowElement.cpp

```diff
@@ -54,6 +54,9 @@ Ref<MathMLRowElement> MathMLRowElement::create(const QualifiedName& tagName, Doc
 
 void MathMLRowElement::childrenChanged(const ChildChange& change)
 {
+    // This does not change the behaviour for valid MathMLFractionElement/MathMLScriptsElement, which inherit this.
```

Comment:
nit: it might be observable if that makes the code slower.

Anyway, can you please just open a bug in bugzilla where you put the details, and just make this a a FIXME like this:

// FIXME: Avoid this invalidation for valid MathMLFractionElement/MathMLScriptsElement  (http://wkbug/XXXXX)

Reply:
Added